Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1632,6 +1636,16 @@ extension BrowserViewController: TabToolbarDelegate {
         }
     }
 
+    func setNightMode(enabled: Bool) {
+        self.profile.prefs.setBool(enabled, forKey: PrefsKeys.KeyNightModeStatus)
+        for tab in self.tabManager.tabs {
+            tab.setNightMode(enabled, force: true)
+        }
+        if let appDelegate = UIApplication.sharedApplication().delegate as? AppDelegate {
+            appDelegate.setNightModeBrightness(enabled)
+        }
```

Comment:
This will drop out of moving `setNightModeBrightness` to `NightModeHelper` to 
`NightModeHelper.setNightMode(enabled)`.